compute_tools/src/config.rs

```diff
@@ -197,6 +197,12 @@ pub fn write_postgres_conf(
         writeln!(file, "neon.disable_logical_replication_subscribers=false")?;
     }
 
+    // We need Postgres to send logs to rsyslog so that we can forward them
+    // further to customers' log aggregation systems.
+    if spec.features.contains(&ComputeFeature::PostgresLogsExport) {
+        writeln!(file, "log_destination='stderr,syslog'")?;
+    }
```

Comment:
This is probably not the right place to put it.

Reply:
@lubennikovaav @ololobus where shall we set `log_destination`?

Reply:
I think this is the right place.

This will override the setting that came from spec,
see this code above
```
    // Write the postgresql.conf content from the spec file as is.
    if let Some(conf) = &spec.cluster.postgresql_conf {
        writeln!(file, "{}", conf)?;
    }
```
    
  and this is exactly what we need.
  maybe just mention this in the comment

Reply:
Let me know if I need to mention something else in the comment.